install/installer/cmd/config.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+package cmd
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	"github.com/gitpod-io/gitpod/installer/pkg/config"
+	"github.com/spf13/cobra"
+	"k8s.io/utils/pointer"
+)
+
+var configOpts struct {
+	ConfigFile string
+}
+
+// configCmd represents the validate command
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Perform configuration tasks",
+}
+
+// configFileExists checks if the configuration file is present on disk
+func configFileExists() (*bool, error) {
```

Comment:
Just out of curiosity, what made you return a pointer to Boolean here?

Reply:
Correct. I don't like returning a boolean if it can return an error - it's entirely stylistic, so can see why someone else might not like/use it